src/devices/Bmp280/IBmp280.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace Iot.Device
+{
+    public interface IBmp280
```

Comment:
Why do you need an interface here? I don't think it is very useful to implement it since it would be the only class implementing it anyway so I would just remove this file.